src/components/admin/Login.js

```diff
@@ -23,49 +34,127 @@ export const Login = () => {
   };
 
   const login = async (email, pass) => {
-    logout(setAuth);
+    logout(setUser);
     try {
       setLoading(true);
       if (!email || email.length === 0 || !pass || pass.length === 0) {
-        throw "email or pass not valid";
+        throw new Error("email or pass not valid");
       }
       const res = await axios.post(`${API_URL}/auth/login`, {
         email,
         password: pass,
       });
-      setAuth(res.data.token);
+      setUser({ token: res.data.token, firstName: res.data.firstName });
       setLoading(false);
-      navigate("/add", { replace: true });
+      navigate("/admin/create/events", { replace: true });
     } catch (e) {
-      handleError("error logging in");
+      handleError(e);
       console.log(e);
       setLoading(false);
     }
   };
-  return (
-    <div>
-      <div>
-        <input
-          type="email"
-          name="email"
-          onChange={(event) => setEmail(event.target.value)}
-          className="border"
-        />
-        <input
-          type="password"
-          name="password"
-          onChange={(event) => setPass(event.target.value)}
-          className="border"
-        />
-        <button
-          onClick={async () => await login(email, pass)}
-          disabled={loading}
-        >
-          login
-        </button>
+  if (width > breakpoint) {
```

Comment:
prefer to use tailwind breakpoints for mobile and desktop styling see here: https://tailwindcss.com/docs/responsive-design#customizing-breakpoints